### PR TITLE
WIP adds spanish again

### PR DESCRIPTION
**Pull request by PKPenguin321**
:cl:
add: Slip on a sombrero or a luchador mask to speak the second most galactically common language.
add: Esto es el fin, grande padre.
/:cl:

This time I made sombreros make you speak it (and only it) while equipped (the code for this sucks dick because I don't know if actual procs for what I want to do exist)

I wanna make wrestling masks make you have the option of speaking it as well but again I barely know a thing about language code and the sombrero code is shitty enough as it is

Send help

### CODE REVIEW

**Comment by Jordie0608** [comment]
When are we getting space conquistadors?

**Comment by XDTM** [comment]
You can make a temporary language holder that only has spanish, and swap it with the mob's current language holder.
Drugginess and aphasia should both do exactly this, if you want to check out their code

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -208,6 +208,23 @@
...
+	if(slot == slot_head)
+		user.grant_language(/datum/language/spanish)
+		user.remove_language(/datum/language/common)//i want to temporarily remove all other languages. if theres a clean way to do this please say so
+
+/obj/item/clothing/head/sombrero/dropped(mob/user)
+	..()
+	if(!iscarbon(user))
+		return
+	var/mob/living/carbon/C = user
+	if(C.get_item_by_slot(slot_gloves) == src)
```
Comment (PKPenguin321) [inline code/modules/clothing/head/misc.dm]: slot_head

**Comment by PKPenguin321** [comment]
is there a readme for all the language related procs or something

**Comment by nicbn** [comment]
No creo que sea una buena idea, eso me parece un poco racista, no?

**Comment by PKPenguin321** [comment]
luchador masks already do this but instead of a language its a speech modifier so i think this is ok

**Comment by nicbn** [comment]
No deja de parecer racista

**Comment by nicbn** [comment]
Actually this language having mexico flag as symbol is like triple racist

**Comment by PKPenguin321** [comment]
ok fine

**Comment by PKPenguin321** [comment]
clearly this pr was made with only hatred for mexicans in mind

**Comment by Buggy123** [comment]
Racist? Have you seen #34916?

>>text_lose_indication = "You no longer feel the urge to switch sides midway through a war. Your love of pasta passes."

**Comment by Bombpants** [comment]
But this is a golden PR

**Comment by Repukan** [comment]
Ahogate en el proximo churro q tell chupes nibcn

**Pull request closed without merging**